src/BuildScriptGeneratorCli/Commands/OptionTemplates.cs

```diff
@@ -10,5 +10,6 @@ internal static class OptionTemplates
         public const string Platform = "-l|--platform <name>";
         public const string PlatformVersion = "--language-version <version>";
         public const string Property = "-p|--property <key-value>";
+        public const string ManifestFileDir = "--manifest-file-dir <directory-path>";
```

Comment:
could be shortened to `--manifest-dir` (regardless of shortening the corresponding C# member)